pkg/controller/querier/querier.go

```diff
@@ -31,22 +33,24 @@ var _ ControllerQuerier = new(controllerQuerier)
 //TODO: expand this interface similarly to AgentQuerier
 type ControllerQuerier interface {
 	GetControllerInfo(controllInfo *v1beta1.AntreaControllerInfo, partial bool)
-	GetNetworkPolicyInfoQuerier() querier.ControllerNetworkPolicyInfoQuerier
+	QueryNetworkPolicies(namespace string, podName string) (applied []antreatypes.NetworkPolicy,
+		egress []antreatypes.NetworkPolicy, ingress []antreatypes.NetworkPolicy)
 }
 
 //TODO: implement interface methods
 type controllerQuerier struct {
 	networkPolicyInfoQuerier querier.ControllerNetworkPolicyInfoQuerier
+	internalNetworkPolicyStore storage.Interface
+	appliedToGroupStore storage.Interface
```

Comment:
after looking at this, I do recommend we go with what I suggested earlier: define a new object type and include it in `controllerQuerier`, instead of adding a new method / new methods to `controllerQuerier` directly and having references to the stores directly in `controllerQuerier`.

I think this is what I suggested earlier:
```
	// Create Antrea object storage.
	addressGroupStore := store.NewAddressGroupStore()
	appliedToGroupStore := store.NewAppliedToGroupStore()
	networkPolicyStore := store.NewNetworkPolicyStore()
	networkPolicyController := networkpolicy.NewNetworkPolicyController(client,
		crdClient,
		podInformer,
		namespaceInformer,
		networkPolicyInformer,
		cnpInformer,
		addressGroupStore,
		appliedToGroupStore,
		networkPolicyStore)
    networkPolicyQueryReplier := networkpolicy.NewNetworkPolicyQueryReplier(
		addressGroupStore,
		appliedToGroupStore,
		networkPolicyStore)
	controllerQuerier := querier.NewControllerQuerier(networkPolicyController, networkPolicyQueryReplier, o.config.APIPort)
```

Reply:
I have moved logic into its own file with the new response struct.